### PR TITLE
chore: update dependabot.yml

### PR DESCRIPTION
**Pull request by asmitbm**
Update dependabot.yml file to update dependencies weekly. This will help us review PRs and maintain the dependencies in a neater way.

Also added `ignore` key to ignore the major version upgrades of dependencies (#168, #166, #164) which might cause breaking down of the website. This will also help reduce unnecessary PRs. Security updates are unaffected by this setting. 

More info here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -16,4 +16,7 @@ updates:
   - package-ecosystem: "npm"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```
Comment (TerryHowe) [inline .github/dependabot.yml:22]: Minor and bug versions will have CVE patches, we want the bot for that

**Review by TerryHowe — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (TerryHowe) [reply .github/dependabot.yml:22]: Ah, sorry missed the `ignore` part

**Review by TerryHowe — COMMENTED** [review]
Why are we running versions with old major versions though?

**Comment by asmitbm** [comment]
> Why are we running versions with old major versions though?

Answered that query here: https://github.com/oras-project/oras-www/pull/164#issuecomment-1551356878

**Review by sajayantony — APPROVED** [review]

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**